Assets/Editor/Tests/TestMathUtils.cs

```diff
@@ -810,5 +810,15 @@ internal class TestMathUtils : MathTestUtils {
     Assert.AreEqual(floats[0], floats1000[0]);
     Assert.AreEqual(floats[floats.Length-1], floats1000[floats1000.Length-1]);
   }
+
+  [Test]
```

Comment:
Awesome! It doesn't look like all tests are functional right now, so we're not running them in CI, but I love that people are adding tests :-)